e2e/production-e2e/Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM python:3.8.2-slim-buster as base
+# syntax=docker/dockerfile:1
 
-# 1. Install node12
-RUN apt-get update && apt-get install -y curl && \
-    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
+FROM nikolaik/python-nodejs:python3.7-nodejs12-slim as base
 
-# 2. Install WebKit dependencies
-RUN apt-get install -y libwoff1 \
+# 1. Install WebKit dependencies
+RUN apt-get update && apt-get install -y libwoff1 \
```

Comment:
You should use apt-get install once and fix the lib version. Remove other unnecessary RUN commands. 
After installing packages rm apt-cache!

Reply:
What library I'm supposed to fix a version of?

Reply:
I don't know, maybe we can skip it and update all time, but we need to use 1 command to install all libs and remove apt-cache after that.